src/styles/styles.scss

```diff
@@ -60,11 +69,11 @@ body {
 
   a {
     text-decoration: underline;
-    color: white;
+    color: $color_white;
     margin-right: 5px;
```

Comment:
can we use the spacing vars here?

Reply:
Yeahh i was planning on coming back and applying the remaining variables in a follow up PR, didnt want to get too in the weeds on this one